features/addressing/managing_countries/editing_country_with_provinces.feature

```diff
@@ -0,0 +1,26 @@
+@addressing
+Feature: Editing country with provinces
```

Comment:
I think it is misleading to name it `Editing country with provinces`. IMHO it should something like `Managing provinces of a country`? Or if you are convinced, that it should be a part of editing it should be placed under _editing_country.feature_